docs/docs/api/management-open-api.yaml

```diff
@@ -2110,6 +2158,113 @@ components:
           type: string
           format: uuid
           description: Warehouse ID where the tabular is stored
+    EndpointStatistic:
+      type: object
+      required:
+      - count
+      - http_route
+      - status_code
+      - created_at
+      properties:
+        count:
+          type: integer
+          format: int64
+          description: Number of requests to this endpoint for the current time-slice.
+        created_at:
+          type: string
+          format: date-time
+          description: |-
+            Timestamp at which the datapoint was created in the database.
+
+            This is the exact time at which the current endpoint-status-warehouse combination was called
+            for the first time in the current time-slice.
+        http_route:
+          type: string
+          description: |-
+            The route of the endpoint.
+
+            Format: `METHOD /path/to/endpoint`
+        status_code:
+          type: integer
+          format: int32
+          description: The status code of the response.
+          minimum: 0
+        updated_at:
+          type:
+          - string
+          - 'null'
```

Comment:
Is this OK? Shouldn't it just be a string that is not required?
There are more like this though.